### ISSUE TITLE
Possibility to add temperature /humidity to hue motion

### ISSUE THREAD

**Issue by betonishard** [mariusmotea/diyHue#193]
Hi,

Great work what you have done so far. Really useful. 
I was wondering if it would be possible to add temperature / humidity readings to the ino of the pir /lux sensor you created. Than you would have the ideal sensor (temp/hum/lux/motion). On the internet I thought i saw several motion sensors  (official) already equipped with temperature sensors. 
And also what transistor are you using in your circuit diagram.

Thanks!

**Comment by mariusmotea** [comment]
Hi,

Is there any option to see the temp in Hue apps? Hue Motion Sensor has a temperature sensor, but so far i did not see to have the valued displayed in the app.

**Comment by betonishard** [comment]
@mariusmotea thanks for your reply.

Well I haven't seen it in hue app, it is available for instance when you connect the hue to homekit for instance. I just loged in to check if there was an update but today I already built it. So I use your software in conjunction with temperature, humidity, motion, lux sending out values via MQTT to my broker in combination with node-red and have it pushed to homekit (my girlfriend's iphone, because I am on android).

So it can be closed. Thanks!
